src/webapi/html_elements/input.rs

```diff
@@ -38,4 +38,44 @@ impl InputElement {
             @{self}.value = @{value};
         }
     }
+
+    /// The offset to the start of the selection.
+    /// This attribute only applies when the input is a text, search, url, telephone or password.
+    /// https://html.spec.whatwg.org/#textFieldSelection
```

Comment:
Wrong link, and shouldn't be a doc comment. (This also applies to other methods you've added.)

The correct one for `selection_start` is:

```
// https://html.spec.whatwg.org/#the-input-element:dom-textarea/input-selectionstart
```

(Always directly link to the piece of IDL.)